docs/compliance/index.md

```diff
@@ -0,0 +1,386 @@
+# Automated Foundational Architecture for PCI DSS-Compliant Environments (v1)
+
+### PUBLIC PREVIEW BETA 1
+
+> [EDIT: INSERT "AZURE CYBER BLUEPRINT" PROGRAM OVERVIEW]
+
+## Overview
+
+The Automated Foundational Architecture for PCI DSS-compliant Environments (v1) provides guidance for the deployment of a PCI DSS-compliant Platform-as-a-Service (PaaS) environment suitable for handling sensitive payment card data. It showcases a common reference architecture and is designed to simplify adoption of Microsoft Azure. This foundational architecture (developed jointly with [Avyan Consulting](http://www.avyanconsulting.com/azureservices), a Microsoft MVP partner) illustrates an end-to-end solution to meet the needs of organizations seeking a cloud-based approach to reducing the burden and cost of deployment.
+
+This foundational architecture meets the requirements of stringent Payment Card Industry Data Security Standards (PCI DSS 3.2) for the collection, storage, and retrieval of payment card data. It demonstrates the proper handling of credit card data (including card number, expiration, and verification data) in a secure, compliant multi-tier environment deployed as an end-to-end Azure-based solution.
+
+This architecture is intended to serve as a foundation for customers to adjust to their specific requirements, and should not be used as-is in a production environment. Deploying an application into this environment without modification is not sufficient to completely meet the requirements of a PCI DSS-compliant solution. Please note the following:
+- This foundational architecture provides a baseline to help customers use Microsoft Azure in a PCI DSS-compliant manner.
+- Achieving PCI DSS-compliance requires that an accredited Qualified Security Assessor (QSA) certify a production customer solution.
+- Customers are responsible for conducting appropriate security and compliance reviews of any solution built using this foundational architecture, as requirements may vary based on the specifics of each customer’s implementation and geography.  
+
+## Architectural Diagram
+
+![](images/pci-architectural-diagram.png)
+
```

Comment:
We should have a section that explains the components of the architecture, similar to our RAs. Briefly (1-2 sentence each) summarize the role of each component - WAF, ASE, SQL DB, jumpbox, etc.